ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug Report
+description: Report a bug
+labels:
+  - bug
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: >-
+        Please provide a general description of the bug you're experiencing
```

Comment:
Should we ask them to make sure to indicate the environment where it's occurring, or should that be its own section.

Reply:
That could certainly be it's own question. There's a regular `input` type that just takes one line.